v2/azure-arm.yaml

```diff
@@ -2587,6 +2634,9 @@ objectModelConfiguration:
       DatabricksProperties:
         DatabricksAccessToken:
           $isSecret: true
+      VirtualMachineSshCredentials_STATUS:
+        PrivateKeyData:
```

Comment:
This shouldn't be included in the `STATUS`?

Reply:
Correct.